app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -252,12 +262,9 @@ DonorsChooseDonationController.prototype.postDonation = function(member, project
         try {
           logger.log('verbose', 'dc.requestToken POST user:%s body:%s', 
             donorPhone, body);
-          // Handles when we get 403/Forbidden but steps in when sufficient funds.
-          // if (typeof body !== 'object') {
-          //   self.endChatWithFail(member);
-          //   logger.error('dc.requestToken user:%s invalid body:%s', donorPhone, body);
-          //   return deferred.promise;
-          // }
+          if (typeof body !== 'object') {
+            logger.error('dc.requestToken user:%s invalid JSON:%s', donorPhone, body);
+          }
```

Comment:
Uncommenting this to log the HTML we get back upon errors from DonorsChoose API (which just started a few minutes ago)

<img width="1162" alt="screen shot 2016-08-16 at 1 51 17 pm" src="https://cloud.githubusercontent.com/assets/1236811/17715390/b3caf974-63b8-11e6-966f-22e6f727164c.png">